corehq/apps/app_manager/templates/app_manager/partials/forms/custom_assertions.html

```diff
@@ -1,4 +1,7 @@
 {% load i18n %}
+{% load hq_shared_tags %}
+{% initial_page_data 'custom_assertions' custom_assertions %}
+<script src="{% static 'app_manager/js/custom_assertions.js' %}"></script>
```

Comment:
I mentioned this on slack when I implemented it - it feels a little unorthodox, but bringing all the requirements into this partial makes it really easy to replicate.  Open to feedback if you see a better way.